monitor-openqa_job

```diff
@@ -18,6 +18,9 @@ obs_component="${obs_component:-"package"}"
 obs_package_name="${1:-""}"
 staging_project=${staging_project:-devel:openQA:testing}
 comment_on_obs=${comment_on_obs:-}
+OPENQA_CLI_RETRIES="${OPENQA_CLI_RETRIES:-7}"
```

Comment:
would be a little more consistent with the rest of the script to use lower-case variables here, no?

```suggestion
OPENQA_CLI_RETRIES="${openqa_cli_retries:-7}"
```

Reply:
Effectively this change overrides the upstream default of 0.

Reply:
yep, that is why I suggest to set (the by upstream used variable) `OPENQA_CLI_RETRIES` to the value of (to the script supplied) `openqa_cli_retries`.

Reply:
I included the suggestion